cmd/kyma/alpha/create/module/moduleconfig.go

```diff
@@ -17,6 +17,7 @@ type Config struct {
 	Version           string                     `yaml:"version"`          //required, the version of the Module
 	Channel           string                     `yaml:"channel"`          //required, channel that should be used in the ModuleTemplate
 	ManifestPath      string                     `yaml:"manifest"`         //required, reference to the manifests, must be a relative file name.
+	Mandatory         bool                       `yaml:"mandatory"`        //required, indicates whether the module is mandatory to be installed on all clusters.
```

Comment:
I think we need to indicate that this is by default set to false. Also, in this case, it should be optional not required.

Reply:
Yes I mean, we need to indicate this (that by default it is set to false) in the documentation

Reply:
And no, Mandatory should be required here. because if it is not given than it will default to false

Reply:
The thing is if mandatory is not provided in the moduleconfig, the command will not fail, it will just set it to false. That's why I believe we should add that the default value is false here.